### PR TITLE
Layout fixes for PLC views

### PR DESCRIPTION
**Pull request by joshlory**
Lots of small fixes bundled up into one PR.  One sample fix:

**Bad wrapping of level names**
level-name-before

**Fixed wrapping & current level highlight**
level-name-after

### CODE REVIEW

**Comment by breville** [comment]
lgtm with comments from in-person walkthrough.

**Comment by mehalshah** [comment]
It's a little weird to see multiple orange outlines for levels that are in multiple stages. But it's not a big deal.

image

File: dashboard/app/views/shared/_header_progress.html.haml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 - small_text = @script && (@script.name == "starwars" || @script.name == "starwarsblocks")
 .header_level
   %div.header_level_container
-    = link_to @stage.localized_title, @script, class: ["header_text", ('small_font_on_tablet' if small_text)]
+    - text = @script.professional_learning_course? ? @stage.localized_name : @stage.localized_title
+    = link_to text, @script, class: ["header_text", ('small_font_on_tablet' if small_text)]
```
Comment (mehalshah) [inline dashboard/app/views/shared/_header_progress.html.haml:5]: nit - single quoted string instead of double quoted

**Comment by mehalshah** [comment]
lgtm

File: dashboard/app/views/shared/_header_progress.html.haml (hunk 1 above)
Comment (joshlory) [inline dashboard/app/views/shared/_header_progress.html.haml:5]: Going to leave this as-is for now since this is existing code (just moved to a new line).

**Pull request merged**